webpack/dev.config.js

```diff
@@ -8,7 +8,8 @@ const stringify = require('json-stringify-safe');
 const webpackMerge = require('webpack-merge');
 const webSocket = require('ws');
 
-const port = 3001;
+const port = process.env.PORT || 3001;
+const apiHost = process.env.API_HOST || 'http://localhost:3000';
```

Comment:
@developish - In what cases might the API_HOST be configured differently in your dev environment?

Reply:
oh, with your middleware script... okay.

Reply:
This is used in `npm start` to point the `API_HOST` to http://localhost:4000 allowing you to override responses from your local server. With a small tweak to the config, this could allow for us to start the front-end client pointing to DEV, TST, QAT or even PROD.